tests/test_covar2d_denoiser.py

```diff
@@ -47,15 +79,20 @@ def test_batched_rotcov2d_MSE(sim):
 
     # Calculate the normalized RMSE of the estimated images.
     nrmse_ims = (imgs_denoised - imgs_clean).norm() / imgs_clean.norm()
```

Comment:
Just want to point out that this is not quite the NRMSE. `norm` has a sum under the root, not a mean. Only pointing out because I would expect the refs to be much lower for NRMSE.

Reply:
Wouldn't the constants cancel out in the division ?

 Anyway, I can't take credit for this gem.  Can clean it up if we need to. LMK

Reply:
My mistake! This is all good.